source/guides/guides/continuous-integration.md

```diff
@@ -44,13 +52,13 @@ Depending on which CI provider you use, you may need a config file. You'll want
 
 ## Caching the Cypress Binary
 
-As of Cypress version 3.0, Cypress downloads its binary to the global system cache - on linux that's `~/.cache/Cypress`. Ensuring this cache persists across builds, you can shave minutes off install time by preventing a large binary download.
+As of {% url "Cypress version 3.0" changelog#3-0-0 %}, Cypress downloads its binary to the global system cache - on linux that is `~/.cache/Cypress`. Ensuring this cache persists across builds, you can shave minutes off install time by preventing a large binary download.
```

Comment:
`Minor`

The second sentence reads a wee bit "off"... possibly, might just be me 🤔

Maybe...

```
By ensuring this cache persists across builds you can shave minutes off install time by preventing a large binary download.
```